README.md

```diff
@@ -582,7 +575,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [loadmill.com](https://www.loadmill.com/) - Automatically create API and load tests by analyzing network traffic. Simulate up to 50 concurrent users for up to 60 minutes for free every month.
   * [preflight.com](https://preflight.com) - No-code automated web testing. Record tests on your browser that are resilient to UI changes and run them on Windows machines. Integrate with your CI/CD. The free plan includes 50 test runs per month with video, HTML sessions and more.
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  Unlimited team members, Demo app and unlimited projects, 5,000 snapshots / month.
-  * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
+  * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://opensource.saucelabs.com/)
```

Comment:
This one I am not so sure about anymore, they dont say its free for open source, they seem to have some process for selecting which open source they support? So I am inclined to just remove it.

Reply:
They offer a Free Plan for everyone (2000 credits/week) and free licenses for open-source projects.

Reply:
It says “free accounts for qualifying initiatives” not all. 

Everyone gets free credits weekly though so we should rather highlight their normal free tier and link to main website.